rebar.config

```diff
@@ -11,7 +11,7 @@
             {jiffy,             "0.15.2"},
             {trails,            "2.1.0"},
             {cowboy_swagger,    "2.1.0"},
-            {escalus,           {git, "git://github.com/esl/escalus.git", {ref, "f20bee4"}}},
+            {escalus,           {git, "git://github.com/esl/escalus.git", {branch, "amoc-scenarios-common-parts"}}},
```

Comment:
We should update the branch now once the Escalus PR is merged.

Reply:
It's done.